src/pudl/metadata/fields.py

```diff
@@ -23,11 +23,7 @@
     TECH_CLASSES,
     TECH_DESCRIPTIONS,
 )
-from pudl.metadata.labels import (
-    ESTIMATED_OR_ACTUAL,
-    FUEL_UNITS_EIA,
-    MOMENTARY_INTERRUPTIONS,
```

Comment:
`MOMENTARY_INTERRUPTIONS` isn't required any more because the same information is stored in a coding table, and integrated into FK constraints.